### ISSUE TITLE
Nuxt 3 (bug): `autoImport: true` breaks build & results in tons of vite-warnings: `(unplugin-formkit plugin) Sourcemap is likely to be incorrect`

### ISSUE THREAD

**Issue by BracketJohn** [formkit/unplugin-formkit#1]
**Description**

TLDR: `autoImport: true` results (a) in 1000x warnings and (b) a broken nuxt app.

When adding:
```ts
// file: `~/nuxt.config.ts`
{
  formkit: {
    autoImport: true
  },
}
```

to our `nuxt.config.ts` we get a the following output (a) when running `nuxi build`:
```sh
ℹ vite v4.5.0 building SSR bundle for production...                                                          3:26:27 PM
ℹ ✓ 1834 modules transformed.                                                                                3:26:39 PM
Inspect report generated at /Users/user/repos/clients/company/company-os/.nuxt/analyze/.vite-inspect
(unplugin-formkit plugin) Sourcemap is likely to be incorrect: a plugin (unplugin-formkit) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
 # ... REPEATED ~1000 TIMES
```

In addition the production build is broken: After opening the first page with formkit nodes the app becomes unresponsive (cannot click save anymore, backwards & forwards navigation does not work). In the console we see the following errors:
<img width="947" alt="image" src="https://github.com/formkit/unplugin-formkit/assets/25911230/24d9e198-d64c-4b56-9335-323da0aa6d3d">

The app breaking only happens, once a page with formkit-elements is visited.

**Expected behavior**

- no warnings during build
- no errors in console + form & app still works

**Additional information**

- I have tested this in one of our projects when it first was released & just now with `nuxt 3.8.1` to ensure that the problems persits
- Discord user `alireza9444` reported the same problem before me: https://discord.com/channels/793529058377072650/1157526909580759071
- @justin-schroeder recommended / asked me / them to open this issue here: https://discord.com/channels/793529058377072650/1157526909580759071/1163859409328406608 (:
- Sadly, I cannot provide a reproduction as the project where it happens is quite large (40-50k LoC) & for a customer of ours, so it's not easy to "narrow down" to a reproduction

**Comment by Archetipo95** [comment]
Same here and it's kinda annoying when running tests

**Comment by justin-schroeder** [comment]
@BracketJohn if you update to `unplugin-formkit@0.2.8` it should resolve the sourcemap warnings. I’m still having a hard time reproducing "breaking" changes — I’ve attempted to blindly add some de-optimizations which may fix it, but if you could test on your project or produce anything resembling a reproduction it would be much appreciated.

**Comment by justin-schroeder** [comment]
More info @BracketJohn: `^0.2.9` is now the version in the `@next` tag of the `@formkit/nuxt`.

**Comment by BracketJohn** [comment]
Hey @justin-schroeder 👋 

Thanks for getting back, this means a lot. I just checked out these changes -> the warnings are now gone & the forms also work 🎊 

One more concrete thing that was detected as breaking by our tests: `getNode('customerForm')` returns `undefined` when having `autoImport: true` and returns the actual form when having `autoImport: false`. In this concrete case we use `getNode` when the user clicks a button to add some data to a form (that is visible & mounted when the user clicks the button).
